installer/bundle/bundle_skel.sh

```diff
@@ -681,8 +680,11 @@ if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
             rmdir /etc/opt /var/opt > /dev/null 2> /dev/null || true
         fi
     fi
-    if [ -f ${COLLECTD_DIR}/oms.conf ]; then
-        rm ${COLLECTD_DIR}/oms.conf > /dev/null 2> /dev/null
+    if [ -f /etc/collectd.d/oms.conf ]; then
+        rm /etc/collectd.d/oms.conf > /dev/null 2> /dev/null
+    fi
+    if [ -f /etc/collectd/collectd.conf.d/oms.conf ]; then
+        rm /etc/collectd/collectd.conf.d/oms.conf > /dev/null 2> /dev/null
```

Comment:
You could just replace all this with two lines:

```
rm -f /etc/collectd.d/oms.conf > /dev/null 2> /dev/null
rm -f /etc/collectd/collectd.conf.d/oms.conf > /dev/null 2> /dev/null
```

The -f says force remove, and don't consider it an error if the file doesn't exist. That eliminates the pre-checks.